applications/matter_bridge/CMakeLists.txt

```diff
@@ -47,13 +47,13 @@ target_sources(app PRIVATE
     ${COMMON_ROOT}/src/bridge/matter_bridged_device.cpp
     ${COMMON_ROOT}/src/bridge/bridge_storage_manager.cpp
     ${COMMON_ROOT}/src/bridge/bridged_device_data_provider.cpp
-    ${COMMON_ROOT}/src/bridge/binding_handler.cpp
+    ${COMMON_ROOT}/src/binding/binding_handler.cpp
     src/zap-generated/IMClusterCommandHandler.cpp
     src/zap-generated/callback-stub.cpp
-    ${COMMON_ROOT}/src/led_widget.cpp
-    ${COMMON_ROOT}/src/persistent_storage_util.cpp
-    ${COMMON_ROOT}/src/task_executor.cpp
-    ${COMMON_ROOT}/src/board.cpp
+    ${COMMON_ROOT}/src/led/led_widget.cpp
+    ${COMMON_ROOT}/src/ps_storage/persistent_storage_util.cpp
+    ${COMMON_ROOT}/src/tasks/task_executor.cpp
+    ${COMMON_ROOT}/src/board/board.cpp
```

Comment:
These changes should be added to the [samples: matter: common: Cleaned up samples common directory](https://github.com/nrfconnect/sdk-nrf/pull/13485/commits/4b0337997497c09ea2ce1d1cc3e80ea87100edbe) commit because they are not related to common binding manager.

In this situation, there is a risk that someone may checkout only the "clean up samples" commit and the bridge won't work. This is not so important, because the main branch is experimental anyway and customers should use main releases, but still, if we can do something to keep our devices working commit by commit we should do it. :)